### DISCUSSION TITLE
Hey there, any recommendations for a fresh and snazzy boilerplate?

### DISCUSSION THREAD

**Discussion by jcenlo** [fastify/fastify#5502] [General] [answered]
First of all, thank you for the help. I’m currently looking for an alternative to Express and stumbled upon Fastify. All the documentation and videos have really sold me on it, and I want to use it. However, I’m encountering a lot of somewhat outdated information. Most YouTube videos, blogs, etc., are about three years old on average. Of course, there are new ones, but they seem like tutorials made on top of other tutorials...

This gives me a sense of a lack of resources and a feeling of abandonment or disinterest (I’m not criticizing, but I think it’s valuable to share these kinds of sensations from new users). I really don’t want to resort to GitHub or Discord to ask many questions since I’m just starting to study it.

I miss having some updated educational resources. I really like the project, but it’s giving me quite a headache.

If you know of any updated material on:

Fastify courses
TypeScript + Postgres boilerplates (with Pino or Sequelize)
I would greatly appreciate it if you could share them.

Thank you very much!

**Comment by metcoder95** [comment, marked as answer]
Maybe this aligns with: https://github.com/fastify/help/issues/1001?

**Reply by jcenlo** [reply]
Thanks i will keep follow that post , is so interesting

**Comment by mcollina** [comment]
The most official learning material for Fastify is https://www.packtpub.com/product/accelerating-server-side-development-with-fastify/9781800563582, and it's listed in https://fastify.dev/resources/.

I know a course on Egghead is in the works, too.

I'll publish a webinar with the material I have ready soon, thanks for letting me know.
